.github/workflows/format-code.yml

```
@@ -6,6 +6,7 @@ on:
 
 jobs:
   format-code:
+    if: ${{ github.actor != 'dependabot[bot]' }}
```

For info - you don't need the `${{ ... }}` any more for certain fields.  Either way works though.
```suggestion
    if: github.actor != 'dependabot[bot]'
```